css/style.css

```diff
@@ -69,3 +68,45 @@ body {
   justify-content: space-between;
   margin: 0 206px;
 }
+/* footer */
+.footer {
+  background: #10367c;
+  height: 630px;
+}
+
+.footer-about {
+  display: flex;
+}
+
+.about {
+  margin: 120px 180px 120px 120px;
+}
+
+.property {
+  margin: 120px 284px 0 0;
+}
+
+li {
+  list-style: none;
+  margin-bottom: 24px;
+  font-size: 20px;
+  line-height: 23px;
```

Comment:
This is **a bad practice**, to use absolute values for line-height
For example if you need to change font-size for mobiles - `line-height: 24px` can look badly
Using of relative values will be better
```
line-height: 2.5;
line-height: 3em;
line-height: 150%;
``` 